ui/ops/src/routes/ops/reports/Reports.vue

```diff
@@ -0,0 +1,95 @@
+<script setup>
+import {timestampToDate} from '@/api/convpb.js';
+import {getDownloadReportUrl, listReports} from '@/api/ui/reports.js';
+import ContentCard from '@/components/ContentCard.vue';
+import {useCohortStore} from '@/stores/cohort.js';
+import {useUiConfigStore} from '@/stores/uiConfig.js';
+import {computed, onMounted, ref} from 'vue';
+
+const reports = ref([]);
+
+const props = defineProps({
+  source: {
+    type: String,
+    default: ''
+  }
+});
+const uiConfig = useUiConfigStore();
+const cohort = useCohortStore();
+const name = computed(() => props.source || (uiConfig.config?.ops?.reports?.source ?? cohort.hubNode?.name ?? ''));
+
+const fetchReports = async () => {
+  const req = { name: name.value}
+  try {
+    const response = await listReports(req);
+    reports.value = response.reportsList.map(report => ({
+      id: report.id,
+      title: report.title,
+      description: report.description,
+      createTime: report.createTime
+    }));
+  } catch (error) {
+    console.error('Failed to fetch reports:', error);
+  }
+};
+
+onMounted(() => {
+  fetchReports();
+});
```

Comment:
This component is not reactive. You should at least refetch reports if the source changes, I think we have some utilities for that somewhere.